src/components/Navigation/Navigation.scss

```diff
@@ -93,8 +112,15 @@ $disabled-fade: 0.6;
     --p-surface-primary-selected,
     $item-selected-background
   );
+
+  .Navigation-newDesignLanguage & {
+    &::before {
+      transform: translateX(0);
+    }
+  }
+
   @media (-ms-high-contrast: active) {
-    background-color: ms-high-contrast-color('selected-text-background');
+    outline: 1px solid ms-high-contrast-color('text');
```

Comment:
This wasn't working before. As @dleroux mentioned yesterday, chromium edge hijacks background color completely. I've been defaulting to outlines for buttons / tabs / checkboxes etc...

Reply:
I think you might be able to remove this altogether and just leave the default?